volumeRendering/main.js

```diff
@@ -4,6 +4,11 @@ var VolumeRenderingPlugin = class VolumeRenderingPlugin extends OHIF.plugins.Vie
 
         this.description = "VolumeRendering OHIF Plugin";
 
+        this.controllerWidget = vtk.Interaction.UI.vtkVolumeController.newInstance({
```

Comment:
This should be hidden by default.

Why is this in this PR? Keep the PRs to one change. If this PR is only about Pan and Zoom, it should only include Pan and Zoom, not VR changes.